cdk/lib/constructs/api.ts

```diff
@@ -25,13 +31,30 @@ export class BackendApi extends cdk.Construct {
       }
     );
 
+    const apiPolicy = new PolicyDocument({
+      statements: [
+        new PolicyStatement({
+          effect: Effect.ALLOW,
+          actions: ["execute-api:Invoke"],
+          resources: ["execute-api:/prod/*/*"],
```

Comment:
This policy should only apply to `execute-api:/prod/ingestapi/*` right? We don't want to apply this same Resource Policy to the public api.